openshift/template.yaml

```diff
@@ -207,6 +208,18 @@ parameters:
   name: IMAGE_TAG
   value: "latest"
 
+- description: Main file to execute
+  displayName: Main file
+  required: true
+  name: MAIN_FILE
```

Comment:
Can you rename it as ENTRY_POINT?

Reply:
`Parameters` don't work in Kubernetes .. can't we get rid of them..?

Reply:
ok

Reply:
I don't think you can easily replace now. I suggest to go with the existing approach until we finalise the templating tool for the the rest of the yamls.